src/components/annotation-summary/annotation-summary.js

```diff
@@ -32,8 +32,8 @@ export const AnnotationSummary = ({ annotationName }) => {
               previousAnnotations && (
                 <Fragment>
                   <span className="annotation-count">{ previousAnnotations.positive + previousAnnotations.negative + currentAnnotations.positive + currentAnnotations.negative }</span> images{` `}
-                  ( <span className="annotation-count positive">{previousAnnotations.positive }</span> positive,{` `}
-                    <span className="annotation-count negative">{previousAnnotations.negative }</span> negative )<br />
+                  ( <span className="annotation-count positive">{ previousAnnotations.positive + currentAnnotations.positive }</span> positive,{` `}
+                    <span className="annotation-count negative">{ previousAnnotations.negative + currentAnnotations.positive }</span> negative )<br />
```

Comment:
@mbwatson I believe I caught a copy and paste error - it should be ```currentAnnotations.negative``` on line 36

Reply:
wow, yep!

new rule: no commits until my morning coffee from now on.